03-Export-Tags-Authors-Stories.py

```diff
@@ -42,18 +42,21 @@ def write_csv(filename, columns):
   if args.archive_type == 'AA':
     author_table = '{0}.authors'.format(args.temp_db_database)
     stories_table = '{0}.stories'.format(args.temp_db_database)
+    item_authors_table = '{0}.item_authors'.format(args.temp_db_database)
     author_name = 'name'
     story_id = 'id'
     story_author_col = 'author_id'
     story_coauthor_col = 'coauthor_id'
     author_id = 'id'
-
+    ia_author_col = 'author_id'
+    ia_story_col = 'item_id'
```

Comment:
I guess this could be `ia_item_col` just for consistency with what it's actually called (same on line 77).